clamscan/clamscan.c

```diff
@@ -216,6 +219,7 @@ void help(void)
     mprintf("    --infected            -i             Only print infected files\n");
     mprintf("    --suppress-ok-results -o             Skip printing OK files\n");
     mprintf("    --bell                               Sound bell on virus detection\n");
+    mprintf("    --jobs=#n             -j             Number of parallel jobs to spawn\n");
```

Comment:
clamdsscan already has the option `-m`/`--multiscan`. Does it make sense to reuse that as the option? To converge the interfaces.

Reply:
I would also be interested in re-using the -m / --multiscan nomenclature to align the clamdscan and clamscan interfaces.